### ISSUE TITLE
Cut a new release of Cheshire

### ISSUE THREAD

**Issue by rymndhng** [dakrone/cheshire#148]
Hey! Is it possible to cut a new release of `cheshire`. It'd be nice to have `parse-stream-strict` available for clj-http to explore solutions for https://github.com/dakrone/clj-http/issues/489. 

I have a branch that's waiting on this function :)

**Comment by dakrone** [comment]
Done in 4539cfd0cf2451e53f4fd2eb18cadd24627b613d :)
